build.zig.zon

```diff
@@ -3,10 +3,14 @@
 	.version = "0.0.0",
 	.paths = .{""},
 	.dependencies = .{
+		.deps_local = .{
+			.url = "file:../Cubyz-libs/zig-out.tar.gz", // macos has no official deps release
```

Comment:
This does not work though. It seems zig fetches the packages independently of the target, giving me an error if the path doesn't exist.

Reply:
I thought it was lazy by default. Added laziness now.